src/components/NavBar.stories.tsx

```diff
@@ -29,24 +29,30 @@ const StyledWrapper = styled.div`
   height: 100%
 `;
 
+const StyledNavBar = styled(NavBar)`
+  position: fixed;
+  left: 2rem;
+  top: 2rem;
+  width: calc(100% - 36rem);
+`;
```

Comment:
why was it necessary to add styling for this that wasn't there already?

Reply:
I couldn't get ladle to make a layout that made sense. They have `main` a few layers in and there's no way to change that, or to give it an id of "root".